src/app/qgsmaptoolregularpolygoncentercorner.h

```diff
@@ -0,0 +1,34 @@
+#ifndef QGSMAPTOOLREGULARPOLYGONCENTERCORNER_H
+#define QGSMAPTOOLREGULARPOLYGONCENTERCORNER_H
+
+/***************************************************************************
+    qgsmaptoolregularpolygoncentercorner.h  -  map tool for adding regular
+    polygon from center and a corner
+    ---------------------
+    begin                : July 2017
+    copyright            : (C) 2017 by Loïc Bartoletti
+    email                : lbartoletti at tuxfamily dot org
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
```

Comment:
Header should be on top of the file... But maybe it doesn't matter?

Reply:
done for all headers